content/concepts/circuit-relay.md

```diff
@@ -1,62 +0,0 @@
----
-title: Circuit Relay
```

Comment:
Are we intentionally deleting this file?

Reply:
Ah no, I deleted this in the UI thinking that it would just remove it from the PR, will re-do